tech-far-hub/gatsby-node.ts

```diff
@@ -0,0 +1,86 @@
+import type { GatsbyNode } from "gatsby";
+import path from "path";
+
+interface ITemplatedNode {
+  id: string;
+  parent: {
+    id: string;
+    name: string;
+    base: string;
+    relativeDirectory: string;
+    relativePath: string;
+  };
+  frontmatter: {
+    slug?: string;
+  };
+  internal: {
+    contentFilePath: string;
+  };
+}
+
+interface ITemplateNodeResultSet {
+  allMdx: {
+    edges: ReadonlyArray<{ readonly node: ITemplatedNode }>;
+  };
+}
+
+interface IGraphQLTemplateNodeResult {
+  errors?: any;
+  data?: ITemplateNodeResultSet;
+}
+
+export const createPages: GatsbyNode["createPages"] = async ({ graphql, actions, reporter }) => {
```

Comment:
Does this override the standard createPages or is this required regardless?

Reply:
I _think_ the way it works is it is yet another createPages, like I had my own plugin.